README.md

```diff
@@ -1,6 +1,6 @@
 # node-oracledb version 1.4
 
-## <a name="about"></a> 1. About node-oracledb
+## <a name="about"></a> About node-oracledb
```

Comment:
The section numbers are very useful when pointing people to doc, e.g. in a phone call.  I would keep them.